services/index/invalidtx.go

```diff
@@ -0,0 +1,170 @@
+package index
+
+import (
+	"fmt"
+	"github.com/Qitmeer/qitmeer/common/hash"
+	"github.com/Qitmeer/qitmeer/core/types"
+	"github.com/Qitmeer/qitmeer/database"
+)
+
+var (
+	itxIndexKey             = []byte("itxbyhashidx")
+	itxidByTxhashBucketName = []byte("itxidbytxhash")
```

Comment:
The naming is too subtle to tell the difference with 
```golang
	txIndexKey = []byte("txbyhashidx")
	txidByTxhashBucketName = []byte("txidbytxhash")
```
to prevent future confusion , better to rename to 
```golang
	invalidTxIndexKey             = []byte("invalid_txbyhashidx")
	invalidTxidByTxhashBucketName = []byte("invalid_txidbytxhash")
```